### ISSUE TITLE
Feature Request: Fire an event once snapshot is complete

### ISSUE THREAD

**Issue by creack** [home-assistant/hassio#1062]
Bump of #839.

After spending some time looking around, looks like the current consensus is to use the `delay` action, which is... less than ideal. Having a proper event fired when the snapshot is done would make a lot of things much more reliable.

My use-case is to upload the snapshot to dropbox/google drive after the snapshot is available.

**Comment by sabeechen** [comment]
Perhaps I could instead recommend [this add-on](https://github.com/sabeechen/hassio-google-drive-backup)?  It doesn't do what you're asking for, but it does manage the work of putting snapshots into Google Drive reliably.

**Comment by creack** [comment]
While this would work indeed, I still think we should have an event. After a quick look at the code, it looks like the plugin sets up a file system watcher to work around the lack of event.
This seems overly complicated and prone to errors while a simple event after snapshot would fit quite well into the Home-assistant pattern and simplify things a lot.

**Comment by cap60552** [comment]
I would like this as well, the "upload snapshots to dropbox" add-on could also benefit from this functionality.

**Comment by CodeFinder2** [comment]
I am interested in this feature as well (for snapshot cleanup/upload).

**Comment by e2m32** [comment]
Adding a me too.

**Comment by p4p3r** [comment]
I'm interested too

**Comment by jfisbein** [comment]
I'm interested too

**Comment by HBDK** [comment]
Yeah this would really be nice to have...

**Comment by smiller171** [comment]
Looks like we need a new issue since this one has been closed as stale
